File: Previous Contest/3560. Find Minimum Log Transportation Cost/Solution.py
class Solution:
    # Time Complexity: O(1)
    # Space Complexity: O(1)
    def calculateCost(self, logLen, maxLoad):
        minCost = 0
        if logLen > maxLoad:
            minCost = (logLen - maxLoad) * maxLoad
        return minCost

    def minCuttingCost(self, n: int, m: int, k: int) -> int:
        s = Solution()
        return s.calculateCost(n, k) + s.calculateCost(m, k)

File: Previous Contest/3560. Find Minimum Log Transportation Cost/Solution.java
class Solution {
    // Time Complexity: O(1)
    // Space Complexity: O(1)
    private long calculateCost(long logLen, long maxLoad) {
        long minCost = 0;
        if (logLen > maxLoad) minCost = (logLen - maxLoad) * maxLoad;
        return minCost;
    }
    public long minCuttingCost(int n, int m, int k) {
        return calculateCost(m, k) + calculateCost(n, k);
    }
}